README.md

```diff
@@ -44,10 +44,18 @@ Read more about this in the [Lumberjack](#Lumberjack) API section.
 ```javascript
 import Lumberjack from '@americanexpress/lumberjack';
 
+let formatter = null;
+// how you format your message can look like:
```

Comment:
>how you format your message can look like
_might_ be better phrased as
>your formatter function can be as simple as
?

Paired with 
>or can be formatted as JSON
changing to
>or can be more complex, like stringifying as JSON
though "stringifying" isn't really a word (though, nor is "stringify")

Reply:
Good suggestions, I've updated the comments with your input in 9e1623c